ja_jp/settings-account.adoc

```diff
@@ -0,0 +1,271 @@
+
+これらの `PST` は {product-short} のデスクトップアプリに <<mail-localstorage.adoc#_ローカルストレージ, ローカルストレージ>> のフォルダへインポートすることが可能です。
+以下の手順では、該当の `PST` ファイルは既にコンピュータ上にコピーしていることを前提しています。
+メール、連絡先、およびカレンダーを `PST` ファイルとしてエクスポートやバックアップする場合、https://support.office.com/en-us/article/back-up-your-email-e5845b0b-1aeb-424f-924c-aa1c33b18833[Back up your email] を参照してください。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *アカウント* をクリックし、編集するアカウントを選択します。
+. *Import from Outlook (.pst file)* までスクロールします。
+. *Choose .pst file* をクリックします。
+. インポートする `PST` を選択します。
+. *Import* をクリックします。
+
+NOTE: {product-short}が`PST`をインポートしているあいだ、ローカルフォルダにアクセスできません。
+*メール* は引き続き使用できますが、*カレンダー* と *連絡先* は使用できません。インポートが完了すると{product-short}が通知します。
+endif::Desktop_app[]
+
+==== データをエクスポートする
+すべてのメール、連絡先、およびカレンダーを `.tgz` ファイルとしてエクスポートすることが可能です。
+特定のフォルダ、メールメッセージ、連絡先、またはカレンダーをエクスポートする場合、それらのアイテムを右クリックし、メニューから *エクスポート* を選択します。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *アカウント* をクリックし、編集するアカウントを選択します。
+. 画面をスクロールし、btn:[エクスポート] ボタンをクリックします。
+
+==== データをインポートする
+`.tgz` ファイルからすべてのメール、連絡先、およびカレンダーをインポートすることが可能です。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *アカウント* をクリックし、編集するアカウントを選択します。
+. *インポート* へスクロールし、btn:[インポート] ボタンをクリックします。
+
+==== モバイルやデスクトップを設定する
+IMAP、 CalDav、 および CardDav はメール、連絡先、カレンダー、およびタスクをモバイルやデスクトップアプリへ同期できるオープンルールです。
+{product-short} ではモバイルのプロファイルを作成し、モバイルデバイスのダウンロード、設定、および同期が可能です。
+
+Android OS は初期から（外部アプリがない状態で）IMAP でメールの管理が可能ですが、CalDAVやCardDAVを利用するためにOpenSyncなどのアプリをインストールする必要があります。
+Mac と iOS では CalDAVやCardDAVを利用できるものがありますので、MacやiOSのデバイスに追加のアプリは必要ありません。
+
+以下にプロファイルのエクスポート方法を案内します。
+モバイルデバイスへこれらのプロファイルをインポートするため、以下のヘルプの内容を参照することを推奨します。
+
+===== プロファイルをエクスポートする
+デバイスと同期するために使用できるプロファイルが５つあります。
+
+メール、カレンダーとタスク、連絡先:: このプロファイルは、モバイルデバイス上のメール、カレンダー、タスク、および連絡先を {product-short}と同期します。
+カレンダーとタスク、連絡先:: このプロファイルは、{product-short} を使って、モバイルデバイス上のカレンダー、タスク、連絡先のみを同期し、メールは同期しません。
+カレンダーとタスク:: このプロファイルは、モバイルデバイス上のカレンダーとタスクのみを{product-short}で同期し、メールや連絡先は同期しません。
+連絡先:: このプロファイルは、モバイルデバイス上の連絡先のみを{product-short}と同期し、メール、カレンダー、タスクは同期しません。
+メール:: このプロファイルでは、{product-short}を使用して、モバイルデバイス上のメールのみを同期し、連絡先、カレンダー、タスクは同期しません。
+
+. image:graphics/cog.svg[cog icon, width=20] -> *設定* を選択します。
+. *アカウント* をクリックし、編集するアカウントを選択します。
+. *携帯またはデスクトップの設定* までスクロールします。
+. ドロップダウンメニューからプロファイルを選択し、 *ダウンロード* をクリックします。
+. ダウンロードするファイルの保存先を指定します。プロファイルを<<_iOSへプロファイルをインポートする, iOS>> や <<_Androidへプロファイルをインポートする, Android>>へインポートする際に、{product-short} はこのファイルを必要とします。
+
+
+===== iOSへプロファイルをインポートする
+IMAP:: https://support.apple.com/en-in/HT201320
+CalDAV:: https://support.apple.com/en-in/guide/iphone/iph3d1110d4/ios
+CardDAV:: https://support.apple.com/en-in/guide/iphone/iph14a87326/ios
+
+===== Androidへプロファイルをインポートする
+
+外部アプリを使わない限り、Androidでは、IMAPのみの対応となりますので、CardDAV、CalDAVについては、お使いのアプリの説明をご確認ください。
+
+==== ActiveSync
+
+. OutlookでActiveSyncを設定するには、このガイドを参照してください :- https://wiki.zimbra.com/wiki/Exchange_ActiveSync%28EAS%29_Outlook_2013

```

Comment:
can we set a text for the link without using `%`?

Reply:
Fixed